### PR TITLE
Adding Workers REST API; models.py: minor modifications

### PR DESCRIPTION
**Pull request by agarwal22megha**
Adding REST API for workers module
Changes in models.py: Removed imports in models.py, appending spaces to make the format consistent throughout the file.

### CODE REVIEW

File: csp/urls.py [hunk 1]
```diff
@@ -22,6 +24,12 @@
 router.register(r'api/category', CategoryViewSet)
 router.register(r'api/module', ModuleViewSet)
 router.register(r'api/projectrequester', ProjectRequesterViewSet)
+router.register(r'api/worker', WorkerViewset)
+router.register(r'api/skill', SkillViewset)
+router.register(r'api/workerSkill', WorkerSkillViewset)
```
Comment (dmorina) [inline csp/urls.py]: use dashes for separating words in urls

File: crowdsourcing/viewsets/worker.py [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+class SkillViewset(viewsets.ModelViewSet):
+    queryset = Skill.objects.all()
+    serializer_class = SkillSerializer
+
+class WorkerSkillViewset(viewsets.ModelViewSet):
+    queryset = WorkerSkill.objects.all()
+    serializer_class = WorkerSkillSerializer
+
+class TaskWorkerViewset(viewsets.ModelViewSet):
```
Comment (dmorina) [inline crowdsourcing/viewsets/worker.py]: does this work after deleting the TaskWorkerSerializer?

File: crowdsourcing/serializers/task.py [hunk 3]
```diff
@@ -1,16 +1,11 @@
...
+class TaskSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Task
+        fields = ('module', 'statuses', 'status', 'deleted', 'created_timestamp', 'last_updated')
+        read_only_fields = ('module', 'statuses', 'created_timestamp', 'last_updated')
 
-
-class TaskWorkerSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = models.TaskWorker
```
Comment (dmorina) [inline crowdsourcing/serializers/task.py]: reason?

File: crowdsourcing/viewsets/worker.py (hunk 2 above)
Comment (agarwal22megha) [inline crowdsourcing/viewsets/worker.py]: I have added the TaskWorkerSerializer in serializers/workers.py

File: csp/urls.py (hunk 1 above)
Comment (agarwal22megha) [inline csp/urls.py]: Done

**Comment by dmorina** [comment]
and resolve merge conflicts please

File: crowdsourcing/serializers/task.py (hunk 3 above)
Comment (agarwal22megha) [inline crowdsourcing/serializers/task.py]: There is no model called TaskProperties in models.py
I have added TaskWorkerSerializer to serializers/workers.py
I'll add other relevant serialisers for tasks once worker is done.

**Comment by agarwal22megha** [comment]
Using PR #221 instead

**Pull request closed without merging**